src/test/java/org/sasanlabs/service/vulnerability/pathTraversal/PathTraversalTest.java

```diff
@@ -0,0 +1,306 @@
+package org.sasanlabs.service.vulnerability.pathTraversal;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.RequestEntity;
+
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class PathTraversalVulnerabilityTest {
+    @InjectMocks
+    private PathTraversalVulnerability pathTraversalVulnerability = new PathTraversalVulnerability();
+    @Test
+    void testGetVulnerablePayloadLevel1WithNullFileName() {
+        Map<String, String> queryParams = new HashMap<>();
+        queryParams.put("fileName", null);
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> response =
+                pathTraversalVulnerability.getVulnerablePayloadLevel1(queryParams);
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+        assertNotNull(response.getBody());
+        assertFalse(response.getBody().getIsValid());
+        assertNull(response.getBody().getContent());
+    }
+    @Test
+    void testGetVulnerablePayloadLevel1() {
+        Map<String, String> queryParams = new HashMap<>();
+        queryParams.put("fileName", "UserInfo.json");
+        ResponseEntity<GenericVulnerabilityResponseBean<String>> response =
+                pathTraversalVulnerability.getVulnerablePayloadLevel1(queryParams);
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+    }
+    @Test
+    void testGetVulnerablePayloadLevel2WithNullFileName() throws URISyntaxException {
```

Comment:
@richard66033 can you please use other conditions for test instead of just Null like `../` is there in level 1 and it passes the test but if ../ is there in second level, it will not allow it. Similarly `..` and `%2f` are not there in level 4 and so on.